tests/conftest.py

```diff
@@ -131,19 +131,19 @@ def upload(self, relativeFilePath, lookupKey = None):
         if lookupKey not in NewContractsFixture.signatures:
             NewContractsFixture.signatures[lookupKey] = NewContractsFixture.generateSignature(resolvedPath)
         signature = NewContractsFixture.signatures[lookupKey]
-        contractAddress = long(hexlify(self.state.evm(compiledCode)), 16)
-        contract = ABIContract(self.state, ContractTranslator(signature), contractAddress)
+        contractAddress = long(hexlify(self.chain.evm(compiledCode)), 16)
+        contract = ABIContract(self.chain, ContractTranslator(signature), contractAddress)
         self.contracts[lookupKey] = contract
         return(contract)
 
     def applySignature(self, signatureName, address):
         assert address
         translator = ContractTranslator(NewContractsFixture.signatures[signatureName])
-        contract = ABIContract(self.state, translator, address)
+        contract = ABIContract(self.chain, translator, address)
         return contract
 
     def resetSnapshot(self):
-        self.state.revert(self.snapshot)
+        self.chain.revert(self.snapshot)
```

Comment:
If the tests don't pass, its possible due to this.  I believe reverting state may have changed in the latest version of pyethereum, I vaguely remember seeing something about it.